.github/workflows/release.yml

```diff
@@ -21,3 +21,6 @@ jobs:
           pvp-bounds: lower
         env:
           HACKAGE_API_KEY: ${{ secrets.HACKAGE_UPLOAD_API_KEY }}
+
+          # The oldest LTS we CI against, to set the lowest lower-bounds we can
+          STACK_YAML: stack-lts-12.26.yaml
```

Comment:
Do you think it makes sense to add a comment to the matrix stanza in the `ci` workflow? If someone where to delete `12.26` from the matrix, they might not know to come over to the `ci` workflow and update this.

Reply:
That's a good note. I was thinking of ways to make it unnecessary.

For example, `stack-upload-action` itself could check for `stack-lts-*.yaml` and use the lowest one automatically.

This would be easy to do and most convenient for us, but it's a bit opinionated, so I'm not sure the best way to do it options-wise for general consumption.